packages/react-theme/package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "@fluentui/eslint-plugin": "*",
+    "@fluentui/react-utilities": "9.0.0-rc.4",
```

Comment:
`no-extraneous-dependencies` identified lint error (previously hidden because the rule didnt work)

`react-utilities` is being used in a [story file](https://github.com/microsoft/fluentui/blob/master/packages/react-theme/src/themes/ColorRamp.stories.tsx#L9). 

would be great if we could do similar STAR approach like we do for eslint-plugin, for such a scenarios.